core/modules/widgets/widget.js

```diff
@@ -401,49 +401,23 @@ Widget.prototype.makeChildWidget = function(parseTreeNode,options) {
 	options = options || {};
 	// Check whether this node type is defined by a custom macro definition
 	var variableDefinitionName = "<$" + parseTreeNode.type + ">";
-	if(parseTreeNode.type !== "transclude" && this.variables[variableDefinitionName] && this.variables[variableDefinitionName].value) {
+	if(!parseTreeNode.isNotRemappable && this.variables[variableDefinitionName] && this.variables[variableDefinitionName].value) {
```

Comment:
hmmm ... double "no". Is there a `isRemappale` flag?

Reply:
Widgets are remappable by default. That means that we'd have to explicitly set `isRemappable` on every parse tree node that we create everywhere across the core. So that's why we choose a flag that is false by default.